.golangci.yml

```diff
@@ -0,0 +1,69 @@
+run:
+  build-tags: 
+    - aws
+    - packet
+    - aks
+    - e2e
+    - baremetal
+    - disruptivee2e
+    - poste2e
+
+  modules-download-mode: vendor
+
+  timeout: 5m
+
+issues:
+  exclude-use-default: false
+  new-from-rev: HEAD~
+  max-same-issues: 0
+  max-issues-per-linter: 0
+
+linters:
+  disable-all: true
```

Comment:
Rather than `disable-all`, could we have a disable list? Or commented out linters in `enable` list, which are disabled, so it is easy to find, which ones are disabled.

Reply:
Cool, will add disabled linters list.